lib/global_config.js

```diff
@@ -157,3 +172,51 @@ const rest = {
   HTTPS_PROTOCOL: 'https'
 };
 exports.rest = rest;
+
+// The default JSON parser
+exports.jsonColumnVariantParser = rawColumnValue => vm.runInContext("(" + rawColumnValue + ")", VM_CONTEXT);
+
+/**
+ * Updates the value of the 'jsonColumnVariantParser' parameter.
+ *
+ * @param {function: (rawColumnValue: string) => any} value
+ */
+exports.setJsonColumnVariantParser = function (value)
+{
+  // validate input
+  Errors.assertInternal(Util.isFunction(value));
+
+  exports.jsonColumnVariantParser = value;
+};
+
+// The default XML parser
+exports.xmlColumnVariantParser = rawColumnValue =>
+{
+  // check if raw string is in XML format
+  // ensure each tag is enclosed and all attributes and elements are valid
+  // XMLValidator.validate returns true if valid, returns an error if invalid
+  var validateResult = XMLValidator.validate(rawColumnValue);
+  if (validateResult === true)
+  {
+    // use XML parser
+    return new XMLParser().parse(rawColumnValue);
+  }
+  else
+  {
+    Logger.getInstance().debug("Unable to parse xml string: %o", validateResult);
+    throw validateResult;
+  }
+};
+
+/**
+ * Updates the value of the 'xmlColumnVariantParser' parameter.
+ *
+ * @param {function: (rawColumnValue: string) => any} value
+ */
+exports.setXmlColumnVariantParser = function (value)
```

Comment:
In changes of SNOW-830291 I've prepared possibility to set custom xmlParserConfiguration:  
/**
 * Set custom xmlParser configuration.
 * Parametrs that you can override:
 *   ignoreAttributes - default true,
 *   attributeNamePrefix - default '@',
 *   attributesGroupName - default null,
 *   alwaysCreateTextNode - default false
 *
 * @param {object} value
 */
Examle:
```
snowflake.configure({
    xmlParserConfiguration: {
        ignoreAttributes: true,
        attributeNamePrefix: '@'
    },
})
``` 
and it is used during creation of new XMLParser
I wonder if it will be useful to have both ways configured? This is more to discuss with @sfc-gh-igarish. When we decide I will implement it in my task